newtab.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>banksy</title>
+    <script src="app.js"></script>
```

Comment:
It's a good idea (and best practice!) to put your script tag at the bottom of the body. This way your script runs only after everything else is loaded. IF script runs without having everything it needs loaded, it could result in errors or unexpected things happening.